### PR TITLE
PR for dap issue #16 section 6.1

### PR DESCRIPTION
**Pull request by JohanBester**
Hi Folks,
I finished work on section 6.1
Kindly please review this PR and let me know what improvements or corrections need to be made here
Thanks

### CODE REVIEW

**Review by coyoteJohn — COMMENTED** [review]
There is a typo at 1289 but it is in an exercise block. Is that for the students to find?
I need to pull this to test the code blocks and I will try to get that done this morning.

File: Section06/6.1-Python_FlowControl.ipynb [hunk 1]
```diff
@@ -48,34 +51,56 @@
...
+    "\n",
+    "Programs are rarely linear.\n",
+    "\n",
+    "Most programs do not work by executing a simple sequential set of statements. The code is constructed so that decisions can be made, and different pathways can be taken through the program, based on changes in variable values.\n",
     "\n",
-    "Simple if\n",
-    "if-else\n",
-    "nested if\n",
-    "if-elif-else\n",
+    "To make this non-sequential flow possible all programming language have a set of control structures which allow this to happen.\n",
```
Comment (jamesleemeredith) [inline Section06/6.1-Python_FlowControl.ipynb:62]: Typo here: Should read "... all programming languages..."

File: Section06/6.1-Python_FlowControl.ipynb [hunk 2]
```diff
@@ -48,34 +51,56 @@
...
     "\n",
+    "The problem with sequential statements is that if the logic has broken in any one of the lines, then the complete source code execution will break, and stop at that point of error.\n",
```
Comment (jamesleemeredith) [inline Section06/6.1-Python_FlowControl.ipynb:64]: Typo here: I believe it should read "... if the logic is broken in any..."

**Review by jamesleemeredith — CHANGES_REQUESTED** [review]
Noticed one or two typos and commented on the appropriate lines.

File: Section06/6.1-Python_FlowControl.ipynb (hunk 1 above)
Comment (JohanBester) [reply Section06/6.1-Python_FlowControl.ipynb]: Thanks James, got that one fixed

File: Section06/6.1-Python_FlowControl.ipynb (hunk 2 above)
Comment (JohanBester) [reply Section06/6.1-Python_FlowControl.ipynb]: Thanks, I got this one as well  :)

**Review by coyoteJohn — APPROVED** [review]
Looks good.

**Review by jamesleemeredith — APPROVED** [review]

**Pull request merged**